### ISSUE TITLE
I have two advices

### ISSUE THREAD

**Issue by nicevlhjfihtqngkldl** [drawdb-io/drawdb#180]
During my trial process, I found the following issues:

1. Comments for fields and tables should not be presented in the form of '--'. It is recommended to use the 'comment' keyword instead.
2. When creating tables in MySQL or MariaDB, it should be allowed to specify the character set encoding and storage engine at the end of the CREATE TABLE statement.

before:
```
/* alipay order */
CREATE OR REPLACE TABLE `alipay_bill` (
	-- table's primary key
	`id` INTEGER NOT NULL AUTO_INCREMENT UNIQUE,
	-- alipay order id
	`alipay_order_id` VARCHAR NOT NULL UNIQUE,
	`my_order_id` INTEGER NOT NULL,
	PRIMARY KEY(`id`, `alipay_order_id`)
);
``` 

what i want is:
```
CREATE OR REPLACE TABLE `alipay_bill` (
	`id` INTEGER NOT NULL AUTO_INCREMENT UNIQUE COMMENT 'table's primary key',
	`alipay_order_id` VARCHAR NOT NULL UNIQUE COMMENT 'alipay's order id',
	`my_order_id` INTEGER NOT NULL,
	PRIMARY KEY(`id`, `alipay_order_id`)
) 
  ENGINE=InnoDB
  DEFAULT CHARSET=utf8mb4
  COLLATE=utf8mb4_general_ci
  COMMENT='alipay order';
```

**Comment by alflanagan** [comment]
1. Recommended by whom, and for what databases? A quick spot check doesn't show the keyword "COMMENT" in Oracle DB, for instance. At the very least, you'd need a configuration option to enable/disable this for older databases.

**Comment by nicevlhjfihtqngkldl** [comment]
> 1. Recommended by whom, and for what databases? A quick spot check doesn't show the keyword "COMMENT" in Oracle DB, for instance. At the very least, you'd need a configuration option to enable/disable this for older databases.

MySQL and MariaDB support the comment keyword. The comment information in the table creation statement will be saved in the database as metadata, which is convenient for subsequent maintainers to understand the meaning of fields and tables. For example, the show create table command can see the table creation statement, which includes the comment information.

**Comment by dhthwy** [comment]
I agree with nicevlhjfihtqngkldl. Comments are important for understanding how things are laid out and for clarity of mind, and as I use this app (in a clock app), this affects me. Thanks for your consideration, looking forward to your reply.
